pycbc/io/live.py

```diff
@@ -195,11 +195,11 @@ def __init__(self, coinc_ifos, ifos, coinc_results, **kwargs):
         # assumptions and checks
         bayestar_check_fields = ('mass1 mass2 mtotal mchirp eta spin1x '
```

Comment:
`bayestar_check_fields` and the comment above should also now be removed, as it is now unused.